test/tools/yulInterpreter/EVMInstructionInterpreter.cpp

```diff
@@ -489,6 +489,10 @@ u256 EVMInstructionInterpreter::eval(
 		yulAssert(false, "Impossible in strict assembly.");
 	case Instruction::DATALOADN:
 		solUnimplemented("DATALOADN unimplemented in yul interpreter.");
+	case Instruction::EOFCREATE:
+		solUnimplemented("EOFCREATE unimplemented in yul interpreter.");
+	case Instruction::RETURNCONTRACT:
+		solUnimplemented("RETURNCONTRACT unimplemented in yul interpreter.");
```

Comment:
It will be simpler to just group them all together:

```suggestion
	case Instruction::DATALOADN:
	case Instruction::EOFCREATE:
	case Instruction::RETURNCONTRACT:
		solUnimplemented("EOF not yet supported by Yul interpreter.");
```

Reply:
done